django_qserializer/serialization.py

```diff
@@ -33,32 +33,32 @@ def __init__(self, *, select_related=None, prefetch_related=None, extra=None):
         else:
             self.extra = {}
 
```

Comment:
Here I made some changes according to the linter I use in my personal projects, `prospector`. I like it because it is very harsh (but can be slow). And also does not have `pyproject.toml` support.

I would suggest to use [ruff](https://github.com/charliermarsh/ruff). I did not use it yet, but Tiangolo uses it (he likes it because is fast).

Or we can use `flake8` as before, but without the pytest integration.